runner.go

```diff
@@ -113,13 +113,28 @@ func (r *Runner) Start() {
 		return
 	}
 
+	var exitCh <-chan int
+
+	// skip the watch phase and spawn child process immediately
+	// if there are no dependencies ie. prefix/secrets found
+	if len(r.dependencies) == 0 {
+		log.Printf("[INFO] (runner) no dependencies to watch")
+		nexitCh, err := r.Run()
+		if err != nil {
+			r.ErrCh <- err
+			return
+		}
+
+		if nexitCh != nil {
+			exitCh = nexitCh
+		}
```

Comment:
This is essentially what `envconsul` runs after the whole `watch` block below. See [here](https://github.com/gocardless/envconsul/blob/aeb809bddde4820f1f4e0dec9bcfa3bac10a9a47/runner.go#L189-L203)